packages/context/src/binding-inspector.ts

```diff
@@ -191,29 +193,36 @@ export type BindingFromClassOptions = {
    * Mapping artifact type to binding key namespaces
    */
   typeNamespaceMapping?: TypeNamespaceMapping;
+  /**
+   * Default scope if the binding does not have an explicit scope
+   */
+  defaultScope?: BindingScope;
 };
 
 /**
  * Create a binding from a class with decorated metadata
  * @param cls A class
```

Comment:
Ditto.

For example:

```suggestion
  * @param cls A class constructor (e.g. a Controller class) or a value provider (Provider class)
```